Modules/_threadmodule.c

```diff
@@ -839,6 +839,11 @@ local_traverse(localobject *self, visitproc visit, void *arg)
     return 0;
 }
 
+#define HEAD_LOCK(runtime) \
```

Comment:
For 3.12+ we should try to reduce duplication of this macro, will see to it in a separate PR.

Reply:
https://github.com/python/cpython/pull/100953